src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -947,14 +947,22 @@ private void startContainer() {
             // Set up permissions on Linux
             String os = System.getProperty("os.name");
             String id = System.getProperty("user.name");
-            if (os != null && os.equalsIgnoreCase("linux") &&
-                id != null && id.equalsIgnoreCase("root")) {
+            if (os != null && os.equalsIgnoreCase("linux")) {
                 // Allow the container server to read the config files e.g. server.xml
-                runCMD("chmod -R o+r " + serverDirectory);
+                runCmd(false, "chmod -R o+r " + serverDirectory);
+                // Allow the container server to read directories like apps and dropins
+                runCmd("chmod -R o+x " + serverDirectory + "/apps " +
+                    serverDirectory + "/dropins " +
+                    serverDirectory + "/configDropins");
```

Comment:
Is the intent to set the o+r on all of the _files_ under serverDirectory, but set o+x on all of its sub-directories only (excluding their files)?  Because the current code would also set o+x on the _files_ inside `apps`, `dropins` etc. which seems excessive.  Should it limit the o+x to only directories e.g. by using `find -type d` as shown [here](https://unix.stackexchange.com/a/431889)?

Reply:
Good point, I wanted to avoid setting x on files for security reasons. I'll use find and see if I can manage the logs directory as well.